cypress.config.js

```diff
@@ -3,6 +3,7 @@ const { defineConfig } = require('cypress');
 module.exports = defineConfig({
   e2e: {
     baseUrl: 'https://www.globalsqa.com/angularJs-protractor/BankingProject/',
+    scrollBehavior: false,
```

Comment:
The `scrollBehavior` property should be a string, such as `'auto'`, `'center'`, `'nearest'`, or `'top'`, instead of a boolean. Please update it to a valid string value.